custom-completions/gh/gh-completions.nu

```diff
@@ -98,12 +98,7 @@ export extern "gh org" [
 ]
 
 def "nu-complete gh pr" [] {
-    ^gh pr --help 
-    | lines 
-    | filter {|line| str starts-with "  " } 
-    | skip 1 
-    | parse "{value}: {description}" 
-    | str trim
+    ^gh pr --help | lines | filter {|line| str starts-with "  " } | skip 1 | parse "{value}: {description}" | str trim
```

Comment:
fwiw, `|line|` is not required in the closure of `filter` :wink: 
unrelated to the changes of this PR but ehe

Reply:
thanks! I will fix it 😄